src/components/views/dialogs/InviteDialog.tsx

```diff
@@ -1249,6 +1262,28 @@ export default class InviteDialog extends React.PureComponent<IInviteDialogProps
         }
     }
 
+    private onDialFormSubmit = ev => {
+        ev.preventDefault();
+        this.transferCall();
+    }
+
+    private onDialChange = ev => {
+        this.setState({dialPadValue: ev.target.value});
```

Comment:
I believe if you're setting `React.FormEvent` as the type of the parameter, TypeScript will complain that you're using `target` over `currentTarget`. As the latter is always guaranteed to be the correct element in all scenarios whether `target` depending on the event type and the context could be different

Reply:
TIL!